Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -12,18 +12,18 @@
   --color-indicator-warning: #DFAF00;
   --color-indicator-success: #4AD439;
   --color-indicator-normal: #23ADF8;
-
   --color-popup-scrollbar-track: #E3E3E3;
   --color-standard-scrollbar-track: #D3D3D3D3;
   --color-standard-background: #FFFFFF;
-
   --remotion-themed-border-radius: 2px;
   --remotion-themed-border-width: 1px;
   --remotion-themed-border: var(--remotion-themed-border-width) solid var(--color-standard-border);
   --remotion-themed-border-focus: var(--remotion-themed-border-width) solid var(--color-primary);
   --remotion-themed-dropdown-scrollbar-border-radius: 10px;
   --remotion-themed-dropdown-scrollbar-width: 10px;
   --remotion-themed-scrollbar-width: 14px;
+  --remotion-themed-input-height: 1.7em;
+  --remotion-themed-padding: 0.3em;
```

Comment:
Please update. I want the input-height to use calc with 2*padding and the lineheight. I think we can make the lineheight a stand-aline value and then use 2*padding+lineheight in the usages further down in Common-CSS. At least, that's how I figure things would work best. If not, please explain.

Reply:
I introduced a new `--remotion-themed-line-height` variable with `1.2em` and used it to calculated `--remotion-themed-min-height` with `--remotion-themed-line-height` _+ 2 *_ `--remotion-themed-padding` (of `0.4em` as mentioned above).